bin/run_epix

```diff
@@ -19,7 +18,7 @@ def pars_args():
     parser.add_argument('--Detector', dest='detector', type=str,
                         action='store', default='XENONnT',
                         help='Detector which should be used. Has to be defined in epix.detectors.')
-    parser.add_argument('--DetectorConfig', dest='detector_config', type=str,
+    parser.add_argument('--EpixConfigOverride', dest='epix_config_override', type=str,
```

Comment:
This was originally replacing `--Config`, to not duplicate WFsim args, is that correct? In that case we should be fine with `--DetectorConfig` now.

Reply:
I replaced it becaus "detector config" is too vague. Like this it's explicit what config you're going to override